demo/demo.js

```diff
@@ -1,20 +1,58 @@
 (function() {
-  angular.module('myApp', ['validation', 'validation.rule', 'ui.bootstrap', 'ui.bootstrap.tpls', 'ui.select', 'ngSanitize'])
+  angular.module('angularValidation', ['validation',
+    'validation.rule',
+    'ui.bootstrap',
+    'ui.bootstrap.tpls',
+    'ui.select',
+    'ngSanitize',
+    'ngCookies',
+    'pascalprecht.translate',
+    'tmh.dynamicLocale'
+  ])
+
+  .constant('LOCALES', {
+    'locales': {
+      'en_US': 'English',
+      'zh_CN': '汉语'
+    },
+    'preferredLocale': 'en_US'
+  })
 
   // -------------------
   // config phase
   // -------------------
-  .config(['$validationProvider', function($validationProvider) {
+  .config(['$validationProvider', '$translateProvider', 'tmhDynamicLocaleProvider', function($validationProvider, $translateProvider, tmhDynamicLocaleProvider, $translate) {
```

Comment:
Actually no. Forgot to remove it.
On Dec 27, 2015 6:07 AM, "Huei Tan" <notifications@github.com> wrote:

> In demo/demo.js
> <https://github.com/huei90/angular-validation/pull/177#discussion_r48455096>
> :
>
> >
> >    // -------------------
> >    // config phase
> >    // -------------------
> > -  .config(['$validationProvider', function($validationProvider) {
> > +  .config(['$validationProvider', '$translateProvider', 'tmhDynamicLocaleProvider', function($validationProvider, $translateProvider, tmhDynamicLocaleProvider, $translate) {
>
> should this $translate need in last param?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/huei90/angular-validation/pull/177/files#r48455096>.
>